content/new-in-22.md

```diff
@@ -169,9 +154,24 @@ Previous changes are listed in the [changelog](https://www.w3.org/TR/WCAG22/#cha
 
 Provide ways to help users navigate, find content, and determine where they are.
 
-### 2.4.11 Focus Not Obscured (Minimum) (AA)
-<p class="persona"><a href="https://www.w3.org/WAI/people-use-web/user-stories/#reporter">Reporter</a> with repetitive stress injury who uses speech recognition software:</p>
+### [AG version] 2.4.11 Focus Not Obscured (Minimum) (AA)
+<section class="brief">
+      <p>In brief:</p>
+      <dl>
+         <dt>Objective</dt><dd>Do not hide the item with focus</dd>
+         <dt>Author task</dt><dd>The item with focus should be at least partially visible in the viewport</dd>
+         <dt>Key beneficiaries</dt><dd>Some users with cognitive disabilities and sighted users reliant on keyboard interaction</dd>
+      </dl>
+  </section>
+
+### [EOWG draft] 2.4.11 Focus Not Obscured (Minimum) (AA)
+In brief:
+<ul>
+  <li><p><span class="whatwhy">What to do: </span>Ensure that when items have focus, they are at least partially visible.</p> </li>
```

Comment:
```suggestion
  <li><p><span class="whatwhy">What to do: </span>Ensure the item receiving keyboard focus is at least partially visible.</p> </li>
```

Reply:
> Ensure the item getting keyboard focus is at least partially visible

It is shorter, though I think would be confusing to some -- "what item are you taking about?"

Remember EOWG proposes the In brief go **before** the SC wording in the Understanding docs &mdash; and that's how we have it in the [What's New doc](https://deploy-preview-196--wai-intro-wcag.netlify.app/standards-guidelines/wcag/new-in-22/#2411-focus-not-obscured-minimum-aa)

Reply:
I can live with it either way. I'm not sure it's any less likely for someone to not have that question in the other phrasing, but can go with your preference

Reply:
OK. We could get other input if worth it? I'll resolve for now and we can re-open if warranted.